### ISSUE TITLE
Annotation based field

### ISSUE THREAD

**Issue by AleMScof** [jreznot/jdl-intellij-plugin#1]
Hi Yuriy!

I've just updated to the latest release and annotation-based errors for entities are gone.
There's another minor bug for the entities field.

This is a sample triggering the error:

```
application {
  config {
    baseName MyApp
    applicationType monolith
    packageName com.test.app
    authenticationType oauth2
    databaseType sql
    devDatabaseType oracle
    prodDatabaseType oracle
    clientFramework angularX
    skipClient false
    skipServer false
    buildTool gradle
    cacheProvider ehcache
    clientPackageManager npm
    clientTheme united
    dtoSuffix DTO
    enableHibernateCache true
    enableSwaggerCodegen true
    enableTranslation true
    entitySuffix false
    jhiPrefix jhi
    nativeLanguage en
    languages [en,de,it]
    messageBroker false
    reactive false
    searchEngine false
    serverPort 8080
    serviceDiscoveryType no
    websocket spring-websocket
    testFrameworks [cypress]
  }
  entities *
}

@dto(mapstruct)
@service(serviceClass)
@paginate(pagination)
@readOnly
@skipClient
entity MyTable (MY_TABLE) {
  @Id customPrimary Long required
  /** some custom comments */
 field1 String
 @fieldNameAsDatabaseColumn(v2012_06)
 v201206 String
}
```
By removing annotations inside entities the plugin is working perfectly,
Regards,
A-

**Comment by jreznot** [comment]
![image](https://user-images.githubusercontent.com/1066152/223545210-8bac7bb7-a6e9-49ac-a52e-0d56e48b5d15.png)

**Comment by jreznot** [comment]
Fixed in 2.2.4
